spec/models/emis_redis/military_information_spec.rb

```diff
@@ -284,4 +284,52 @@
       )
     end
   end
+
+  describe '#service_history' do
+    context 'with one military service episode' do
+      it 'for the episode, it should return the branch of service, start date, and end date' do
+        VCR.use_cassette('emis/get_military_service_episodes/valid') do
+          service_history = [
+            service_history_object(begin_date: '2007-04-01', end_date: '2016-06-01')
+          ]
+
+          expect(subject.service_history.as_json).to eq service_history
+        end
+      end
+    end
+
+    context 'with multiple military service episodes' do
+      it 'for each episode, it should return the branch of service, start date, and end date' do
+        VCR.use_cassette('emis/get_military_service_episodes/valid_multiple_episodes') do
+          service_history = [
+            service_history_object(begin_date: '2007-04-01', end_date: '2016-06-01'),
```

Comment:
This is interesting, since it's a natural usage of the models that we were defining in swagger. It's neat we broke it out into ServiceHistory/ServiceHistoryObject. I almost wonder about `ServiceHistory` being a collection of `ServicePeriods`.

Reply:
@jkassemi - If I am understanding you correctly, wanted to clarify that `service_history_object` is actually just a _spec helper/support method_ I created, down at the bottom of the file :)

Reply:
Yeah, no action needed here, just making an observation on the reasoning and organization that you used to set up the service history model. When someone else is building something consuming this data, it seems to make sense that they'd build something similar, so we'd probably want our swagger spec to mimic this.